src/views/Shipments.vue

```diff
@@ -63,10 +63,16 @@ export default defineComponent({
       const viewSize = vSize ? vSize : process.env.VUE_APP_VIEW_SIZE;
       const viewIndex = vIndex ? vIndex : 0;
       const payload = {
-        viewSize,
-        viewIndex,
-        facilityId: this.user.facilityId,
-        statusId: "PURCH_SHIP_SHIPPED"
+        "inputFields":{
+            "destinationFacilityId": this.user.facilityId,
+            statusId: "PURCH_SHIP_SHIPPED",
+            shipmentTypeId: "INCOMING_SHIPMENT"
+        },
+        "entityName": "Shipment",
```

Comment:
We could use ShipmentAndType now with condition that either shipmentTypeId or parentTypeId as INCOMING_SHIPMENT